app/services/stat_retrieval_service.rb

```diff
@@ -22,6 +22,7 @@ def stats_string(user)
     def rankings(limit: 1000)
       ranked_users = []
       User.where('rank != 1500').order('rank desc').limit(limit).each do |u|
```

Comment:
Also I just realized I can require a minimum number of games to be ranked too! I'll put it to 5.